Berlin/BerHsOr9834.xml

```diff
@@ -68,6 +68,12 @@ schematypens="http://relaxng.org/ns/structure/1.0"?>
                      </decoNote>
                   </decoDesc>
                </physDesc>
+               <history>
+                   <origin/> Ethiopia.
+
+                   <provenance>The manuscript was private posession of <persName ref"PRS5072Hammersc"/>. He had received it as a gift from a local monk on <placeName ref="LOC2422DagaE"/> in <date>1968</date>.
```

Comment:
you need "=" after `ref`
(`<persName ref="PRS5072Hammersc"/>`)

check if autocomplete and validation of your Atom is activated, it should tell you where schema errors are